addons/project/static/src/components/project_right_side_panel/components/project_right_side_panel_section.xml

```diff
@@ -9,7 +9,10 @@
                 </div>
                 <t t-slot="header"/>
             </div>
-            <div class="o_rightpanel_data fs-6" t-if="props.showData">
+            <div class="o_rightpanel_data text-muted mt-2 mb-4 fst-italic" t-if="props.slots.helper and !props.showData">
+                <t t-slot="helper"/>
+            </div>
```

Comment:
I don't think it is a good idea to add a helper slot. The helper could be displayed in the default one instead, no? 🤔